src/sentry/static/sentry/app/views/projectsDashboard/teamMembers.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {Member} from 'app/types';
+import AsyncComponent from 'app/components/asyncComponent';
+import AvatarList from 'app/components/avatar/avatarList';
+
+type Props = AsyncComponent['props'] & {
+  teamId: string;
+  orgId: string;
+};
+
+type State = AsyncComponent['state'] & {
+  members?: Member[];
+};
+
+class TeamMembers extends AsyncComponent<Props, State> {
+  static propTypes = {
```

Comment:
do you still need the propTypes here?

Reply:
I don't think we've been removing them yet

Reply:
we are removing them if they are no longer being used 😉